zerver/openapi/zulip.yaml

```diff
@@ -13437,6 +13448,19 @@ paths:
 
                           **Changes**: None added as an option in Zulip 3.0 (feature level 1)
                           to disable video call UI.
+                      realm_jitsi_server_url:
+                        type: string
+                        nullable: true
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
+
+                          The URL of the custom Jitsi Meet server to use when the
+                          [video call provider](/help/start-a-call) is configured as Jitsi Meet.
+
+                          - null, the default, means the realm will inherit the server-level setting.
+
+                          **Changes**: New in Zulip 8.0 (feature level 210). Previously, this was only
+                          available as a server-level configuration.
```

Comment:
I think this description might be wrong; we have a separate/existing `jitsi_server_url` field (no `realm_` prefix) which is what the client should use to generate a call -- that value is always a URL (It will be the server-level value if this is unset.).

I think the correct description for this is:

> The URL of the custom Jitsi Meet server configured in this organization's settings. Clients should
> NOT use this field when starting video calls; the `jitsi_server_url` property should be used for that.
> This field is only used for the settings UI.
> 
> - `null`, the default, means this realm is configured to use the server's default Jitsi Meet server.

Reply:
Started https://chat.zulip.org/#narrow/stream/378-api-design/topic/realm_jitsi_server_url/near/1637106 to try to figure out what we want to do here; will likely require some minor code changes too once we figure it out.